resources/kubernetes/operator/helm/values.yaml

```diff
@@ -44,65 +50,82 @@ clusterRole:
 # Cluster Name is detected automatically for EKS/GKE/AKS
```

Comment:
>Cluster Name is detected automatically for EKS/GKE/AKS 

Cluster Name is automatically detected for EKS/GKE/AKS, add the below value when cluster name is not available.